x-pack/plugins/enterprise_search/public/applications/app_search/components/relevance_tuning/relevance_tuning_logic.test.ts

```diff
@@ -0,0 +1,296 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { LogicMounter } from '../../../__mocks__';
+
+import { BoostType } from './types';
+
+import { RelevanceTuningLogic } from './relevance_tuning_logic';
+
+describe('RelevanceTuningLogic', () => {
+  const searchSettings = {
+    boosts: {
+      foo: [
+        {
+          type: 'value' as BoostType,
+          factor: 5,
+        },
+      ],
+    },
+    search_fields: {},
+  };
+  const schema = {};
+  const schemaConflicts = {};
+  const relevanceTuningProps = {
+    searchSettings,
+    schema,
+    schemaConflicts,
+  };
+  const searchResults = [{}, {}];
+
+  const { mount } = new LogicMounter(RelevanceTuningLogic);
```

Comment:
[optional organization nit] Let's put destructured dependencies at the top of the describe block and have constants after them (see https://github.com/elastic/kibana/pull/88494#discussion_r558483164) - other logic files now have this order

Reply:
2eb1d37